bots/images/scripts/fedora.setup

```diff
@@ -157,7 +155,10 @@ srpm=$(/var/lib/testvm/make-srpm $TEST_SOURCE)
 #
 # https://bugzilla.redhat.com/show_bug.cgi?id=1447627
 
-su builder -c "/usr/bin/mock --no-bootstrap-chroot --verbose --installdeps \"$PWD/$srpm\""
+# The sed command strips the BuildRequires label and version specifiers, and
+# after adds quotes to packages with brackets in their name.
+su builder -c "/usr/bin/mock --no-bootstrap-chroot --verbose -i $(curl https://raw.githubusercontent.com/cockpit-project/cockpit/master/tools/cockpit.spec |\
+               sed -n '/^BuildRequires:/ {s/^.*: //; s/>=.*$//; s/%{.*}//; s/\([^ ]*(.*)[^ ]*\)/"\1"/; p}' | tr '\n' ' ')"
```

Comment:
This may be a bit dangerous if there are multiple `:` in the line, e. g. in package names or version specifiers. No idea if that can actually happen, but it may be better to use a non-greedy `^.*` variant, or perhaps `/^[^ ]*: /`?